src/fastcs/connections/serial_connection.py

```diff
@@ -0,0 +1,54 @@
+import asyncio
+from dataclasses import dataclass
+
+import aioserial
+
+
+class NotOpenedError(Exception):
+    pass
+
+
+@dataclass
+class SerialConnectionSettings:
+    port: str
+    baud: int = 115200
```

Comment:
It would be nice if this was an enum of valid baud rates. Even better if it could be imported / built from the serial builtin library so we don't need to maintain it.

Reply:
Agreed that it is too difficult to provide intellisense for the possible baud rates without completly restricting the values, which we don't want to do.